packages/ERTP/src/amountMath.js

```diff
@@ -105,15 +105,17 @@ const assertValueGetAssetKind = value => {
  *
  * Made available only for testing, but it is harmless for other uses.
  *
- * @template V
+ * @template {AmountValue} V
  * @param {V} value
  * @returns {MathHelpers<V>}
  */
 export const assertValueGetHelpers = value =>
   // @ts-expect-error cast
   helpers[assertValueGetAssetKind(value)];
 
-/** @type {(allegedBrand: Brand, brand?: Brand) => void} */
+/**
+ * @type {(allegedBrand: Brand<any>, brand?: Brand<any>) => void}
```

Comment:
Brand's type parameter is no longer optional? If I'm reading that right, that's an awfully big DX regression.

Reply:
it still is but the default is `AssetKind`, not `any`. This change was needed only here. I didn't look deeply into why